lib/Confirmed_Ride.dart

```diff
@@ -0,0 +1,347 @@
+import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart' as UrlLauncher;
+
+class ConfirmedRide extends StatefulWidget {
+  @override
+  _ConfirmedRideState createState() => _ConfirmedRideState();
+}
+
+class _ConfirmedRideState extends State<ConfirmedRide> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+        appBar: AppBar(
+          title: Text('Schedule',
+              style: TextStyle(
+                  color: Colors.black, fontSize: 20, fontFamily: 'SFPro')),
```

Comment:
For consistency make a custom widget for the header in the Appbar. For all other texts etc. you do that. I think it makes it more concise and readable.

Reply:
Thanks for the feedback, I will fix these them as well as apply it to other files for consistency.